src/util/languageUtils.js

```diff
@@ -38,16 +38,10 @@ export const getLanguage = (
 	request,
 	supportedLangs,
 	defaultLang = LANG_DEFAULT
-) => {
-	// return the first language hit in the order of preference
-	supportedLangs.sort(l => l !== defaultLang);
```

Comment:
you could, but then you have a redundant `defaultLang` key somewhere deeper in the array that isn't doing anything.

given that the 'verbose' alternative is `supportedLangs.sort(l => l === defaultLang ? -1 : 1);`, I'd prefer 'correct' over terse.